src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```diff
@@ -0,0 +1,42 @@
+<Project>
+
+  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
+    <ProjectCapability Include="UseMauiCore" />
+    <ProjectCapability Condition=" '$(UseMaui)' == 'true' " Include="UseMaui" />
+  </ItemGroup>
+
+  <Import
+      Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' or '$(UseMauiAssets)' == 'true' "
+      Project="Sdk.targets" Sdk="Microsoft.Maui.Sdk-$([MSBuild]::GetTargetFrameworkVersion($(TargetFramework)))"
```

Comment:
This file is imported by every type of .NET project, so `$(TargetFramework)` could be like `net472`, `netstandard2.0`, etc., would it be better to just have an `<Import/>` for 6 and an `<Import/>` for 7?

Reply:
gotcha